src/ripple/app/book/impl/BookTip.cpp

```diff
@@ -46,31 +47,33 @@ BookTip::step ()
     for(;;)
     {
         // See if there's an entry at or worse than current quality.
-        auto const page (
-            view().getNextLedgerIndex (m_book, m_end));
+        auto const first_page (view().getNextLedgerIndex (m_book, m_end));
 
```

Comment:
This is a critical change that we apparently pushed into release... we shouldn't be mixing it with this minor refactoring which doesn't obviously improve the code.